capella_model_explorer/components.py

```diff
@@ -110,7 +111,24 @@ def breadcrumbs(
                         ),
                     )
                 )
-
+    hamburger = (
+        ft.Li(
+            ft.Div(
+                ft.A(
+                    icons.hamburger(),
+                    onclick=(
+                        "console.log('clicked');"
+                        "document.getElementById('template-sidebar').ariaExpanded = true;"
+                        "this.closest('li').classList.add('hidden');"
```

Comment:
Hiding the hamburger button when opening the sidebar also causes a shift in the navbar layout, which can look a bit odd in some cases. Have you tried how it looks (and feels) if you replace the hamburger button with a close button? This would avoid the layout shift, but obviously then the close button wouldn't be in the top-right corner, so I'm not sure sure yet what's best there from a UX perspective.